lib/views/alaveteli_pro/batch_request_authority_searches/_begginer_batch_pop_up.html.erb

```diff
@@ -0,0 +1,53 @@
+<% content_for :javascript_head do %>
+  <%= javascript_importmap_tags("modal", importmap: Rails.application.theme_importmap) %>
+<% end %>
+
+<div class="modal-content modal-hidden batch-request-pop-up"  aria-hidden="true" data-auto-open="true" data-open-delay="100" data-click-outside-close="false">
```

Comment:
This can still be closed with <kbd>Esc</kbd>, which would be my first instinct when seeing a modal pop up. Can we prevent that?